jekyll/_cci2/triggers-overview.adoc

```diff
@@ -55,23 +55,23 @@ TIP: If your code is integrated via **GitHub OAuth or Bitbucket Cloud**, you can
 
 NOTE: Triggering a pipeline via the CircleCI v2 API is not currently supported for GitLab projects.
 
-[.tab.apitrigger.GitHub_App_/_Bitbucket_Data_Center]
+[.tab.trigger-pipeline-api.GitHub_and_Bitbucket]
 --
-For **projects integrated with GitHub App or Bitbucket Data Center**, you can trigger a pipeline for a project using the link:https://circleci.com/docs/api/v2/index.html#tag/Pipeline/operation/triggerPipelineRun[v2 API endpoint] `https://circleci.com/api/v2/project/circleci/<org-id>/<project-id>/pipeline/run`.
+This method is supported for GitHub (OAuth and App) and Bitbucket (Cloud and Data Center) orgs.
+
+TIP: For **projects integrated with GitHub OAuth app or Bitbucket Cloud**, you can also trigger a pipeline using the deprecated link:https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline[Trigger a New Pipeline] endpoint, which is described in the **CircleCI server** tab.
```

Comment:
This is one those functionalities where support is determined by the type of **pipeline** rather than the type of org (org project). 
Can we say 
```suggestion
TIP: **GitHub OAuth and Bitbucket Cloud pipelines** can also be triggered using the deprecated link:https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline[Trigger a New Pipeline] endpoint, which is described in the **CircleCI server** tab.
```

Reply:
Actually, Moses recommended not using the word "deprecated" until we have a clear deprecation plan, in order to avoid lots of support tickets being opened.

```suggestion
TIP: **GitHub OAuth and Bitbucket Cloud pipelines** can also be triggered using the link:https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline[Trigger a New Pipeline] endpoint, which is described in the **CircleCI server** tab.
```

And maybe even adding something like "but we recommend using the _recommended_ one"?